crates/aptos/src/test/mod.rs

```diff
@@ -1114,6 +1114,8 @@ impl CliTestFramework {
         is_multi_step: bool,
     ) -> CliTypedResult<ProposalSubmissionSummary> {
         SubmitProposal {
+            #[cfg(feature = "no-upload-proposal")]
```

Comment:
It means that instead of fetching the proposal from a url we construct the proposal locally from a json file. This line of change is just for fixing tests.